docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -165,15 +170,87 @@ To discard your proposal after confirming the validator was added, call `ibft_di
     ```bash
     curl -X POST --data '{"jsonrpc":"2.0","method":"ibft_discardValidatorVote","params":["0xFE3B557E8Fb62b89F4916B721be55cEb828dBd73"], "id":1}' <JSON-RPC-endpoint:port>
     ```
-### Removing a Validator
+#### Removing a validator
 
 The process for removing a validator is the same as adding a validator except you specify `false` as the second parameter of `ibft_proposeValidatorVote`.
 
-### Epoch Transition
+#### Epoch transition
 
 At each epoch transition, all pending votes collected from received blocks are discarded. Existing proposals remain 
 in effect and validators re-add their vote the next time they create a block. 
 
 An epoch transition occurs every `epochLength` blocks where `epochlength` is defined in the IBFT genesis file.
 
-
+### Adding and removing validators without voting
+
+If network conditions mean voting cannot be used to change validators you can bypass voting and specify new
+validators in the genesis file. For example, a majority of the current validators are no longer
+participating in the network so a vote to add or remove valiators will never be successful. 
+
+!!! caution
+    Do not specify a transition block in the past. Specifying a transition block in the past may result in 
+    unexpected behaviour.  
+
+To add or remove validators without voting:
+
+1. Stop all nodes in the network.
+1. In the genesis file, add the `transitions` configuration item where: 
+
+    * `<BlockNumber>` is the upcoming block at which to change validators.
+    * `<ValidatorAddressX> ... <ValidatorAddressZ>` are the account addresses of the validators after `<BlockNumber>`.
```

Comment:
nit: is it worth mentioning they are _string_ elements representing the account addresses (i.e. define the json type)

Reply:
Probably.  Have updated.

Reply:
Updated